src/reducers/status/index.js

```diff
@@ -0,0 +1,117 @@
+import { handleActions } from 'redux-actions'
+import reduceReducers from 'reduce-reducers'
+
+import {
+    clearLocalAlert,
+    clearGlobalAlert,
+    setMainLoader
+} from '../../actions/status'
+
+const initialState = {
+    mainLoader: false,
+    actionStatus: {},
+    globalAlert: {},
+    localAlert: {}
+}
+
+const alertReducer = (state, { error, ready, payload, meta, type }) => {
+    const actionStatus = {
+        ...state.actionStatus,
+        [type]: typeof ready === 'undefined'
+            ? undefined
+            : {
+                success: typeof ready === 'undefined' 
```

Comment:
this also seems to be almost the same but slightly different than `localAlert` and `globalAlert`. Is there any reason for this?

Reply:
I plan to use `actionStatus` to store basic data for all actions that were dispatched - sometimes we dispatch many actions one after another and it's difficult to track their results. So `actionStatus` stores data for all actions, and `globalAlert/localAlert` stores only data that we want to display in UI. I also use `actionStatus` for checking if action is pending, and in `globalAlert` component for animations.